codec/src/test/java/io/netty/handler/codec/compression/BrotliDecoderTest.java

```diff
@@ -131,7 +131,7 @@ private void testDecompression(final ByteBuf expected, final ByteBuf data) {
         decompressed.release();
     }
 
-  private void testDecompressionOfBatchedFlow(final ByteBuf expected, final ByteBuf data) {
+    private void testDecompressionOfBatchedFlow(final ByteBuf expected, final ByteBuf data) {
```

Comment:
nit: revert changes in this file as this is just formatting changes

Reply:
I inlined it properly. Does it really need reverting?

Reply:
not a must... that said I usually prefer to change only what is really relevant for a change as it takes things easier to review etc.

Reply:
Will take care from next time for sure.